users/faq.rst

```diff
@@ -226,7 +226,21 @@ programs to achieve this such as rsync or Unison.
 When I do have two distinct Syncthing-managed folders on two hosts, how does Syncthing handle moving files between them?
```

Comment:
A more compact title would be "Does Syncthing handle moving files between shared folders?"